lib/buildingsync/model_articulation/building.rb

```diff
@@ -95,6 +96,9 @@ def read_xml(build_element, site_occupancy_type, site_total_floor_area, ns)
       if build_element.attributes['ID']
         @id = build_element.attributes['ID']
       end
+
+      # read location specific values
+      read_location_values(build_element, ns)
       # city and state
       read_city_and_state_name(build_element, ns)
```

Comment:
@TMaile this method call can be removed - covered above.

Reply:
fixed in 22c131d